dask_bigquery/core.py

```diff
@@ -174,3 +196,110 @@ def make_create_read_session_request(row_filter="", max_stream_count=0):
 
         graph = HighLevelGraph({output_name: layer}, {output_name: set()})
         return new_dd_object(graph, output_name, meta, divisions)
+
+
+def to_gbq(
+    df,
+    *,
+    project_id: str,
+    dataset_id: str,
+    table_id: str,
+    gs_bucket: str = "dask-bigquery-tmp",
+    credentials: Credentials = None,
+    delete_bucket: bool = False,
+    parquet_kwargs: dict = None,
+    load_job_kwargs: dict = None,
+):
+    """Write dask dataframe as table using BigQuery Load API.
```

Comment:
I'd maybe call it a load job rather than the load API (the relevant API is just `/jobs` if I remember correctly)